aspnetcore/fundamentals/logging/loggermessage.md

```diff
@@ -17,45 +16,63 @@ By [Luke Latham](https://github.com/guardrex)
 
-The string provided to the `Define` method is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates because they serve as property names within structured log data. The sample app uses and we recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names (for example, "{Count}", "{FirstName}").
-
 ## LoggerMessage.DefineScope
 
 [DefineScope(String)](/dotnet/api/microsoft.extensions.logging.loggermessage.definescope) creates a `Func` delegate for defining a [log scope](xref:fundamentals/logging/index#log-scopes). `DefineScope` overloads permit passing up to three type parameters to a named format string (template).
 
-The string provided to the `DefineScope` method is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates because they serve as property names within structured log data. The sample app uses and we recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names (for example, "{Count}", "{FirstName}").
+## Define and DefineScope
+
+The string provided to the `Define` and `DefineScope` methods is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates.  Placeholder names should be descriptive and consistent because they serve as property names within structured log data. We recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names. For example, `"{Count}", "{FirstName}"`.
 
 ## Implementing LoggerMessage.Define
 
 Each logging message is an `Action` held in a static field created by `LoggerMessage.Define`. For example, the sample app creates a field to describe a log message for a GET request for the Index page (*Internal/LoggerExtensions.cs*):
 
 [!code-csharp[Main](loggermessage/sample/Internal/LoggerExtensions.cs?name=snippet1)]
 
-For the `Action`, specify a log level, a unique event identifier (`EventId`) with the name of the static extension method, and the message template. For a request for the Index page of the sample app, the log level is `Information`, the event id is `1` with the name of the `IndexPageRequested` method, and the template is a string:
+For the `Action`, specify:
+
+* The log level.
+* A unique event identifier (`EventId`) with the name of the static extension method.
+* The message template. 
+
+A a request for the Index page of the sample app:
```

Comment:
"A a"

... and the list that follows doesn't form complete sentences ... only fragments. How about ...

> A request for the Index page of the sample app sets the:
> \* log level to \`Information\`.
> \* event id to \`1\` with the name of the \`IndexPageRequested\` method.
> \* template as a string.